packages/horizon/contracts/data-service/utilities/ProvisionManager.sol

```diff
@@ -23,6 +23,14 @@ import { ProvisionManagerV1Storage } from "./ProvisionManagerStorage.sol";
 abstract contract ProvisionManager is Initializable, GraphDirectory, ProvisionManagerV1Storage {
     using UintRange for uint256;
 
+    // Constants
+    uint32 private constant DEFAULT_MIN_VERIFIER_CUT = type(uint32).min;
+    uint32 internal constant DEFAULT_MAX_VERIFIER_CUT = uint32(PPMMath.MAX_PPM);
+    uint64 private constant DEFAULT_MIN_THAWING_PERIOD = type(uint64).min;
+    uint64 internal constant DEFAULT_MAX_THAWING_PERIOD = type(uint64).max;
+    uint256 private constant DEFAULT_MIN_PROVISION_TOKENS = type(uint256).min;
+    uint256 internal constant DEFAULT_MAX_PROVISION_TOKENS = type(uint256).max;
```

Comment:
good call making these internal so they can be inherited.
I think we should make all internal, in this case the SubgraphService needed those 3 but other data services might need to use the rest so it's better if we make them available.